apis/v1alpha2/gateway_types.go

```diff
@@ -229,6 +233,9 @@ const (
 
 	// This reason is used with the "Accepted" condition when no controller has
 	// reconciled the Gateway.
+	GatewayReasonPending GatewayConditionReason = "Pending"
```

Comment:
Nit, but I think this should be updated to match the Listener documentation to note it should also be used with the "Ready" condition when the status is "Unknown".